specification/servicelinker/ServiceLinker.Management/models.tsp

```diff
@@ -0,0 +1,1463 @@
+   */
+  secretType?: SecretSourceType;
+
+  /**
+   * Deprecated, please use #/definitions/DaprConfigurationList instead
+   */
+  daprProperties?: DaprProperties;
+
+  /**
+   * The configuration names to be set in compute service environment.
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/missing-x-ms-identifiers" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  names?: ConfigurationName[];
+}
+
+/**
+ * The configuration names.
+ */
+model ConfigurationName {
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  value?: string;
+
+  /**
+   * Description for the configuration name.
+   */
+  description?: string;
+
+  /**
+   * Represent the configuration is required or not
+   */
+  required?: boolean;
+}
+
+/**
+ * Dapr configuration list supported by Service Connector
+ */
+model DaprConfigurationList is Azure.Core.Page<DaprConfigurationResource>;
+
+/**
+ * Represent one resource of the dapr configuration list
+ */
+model DaprConfigurationResource {
+  /**
+   * The properties of the dapr configuration.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-private-usage" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  properties?: DaprConfigurationProperties;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+model DaprConfigurationProperties {
+  /**
+   * Supported target resource type, extract from resource id, uppercase
+   */
+  targetType?: string;
+
+  /**
+   * The authentication type.
+   */
+  authType?: AuthType;
+
+  /**
+   * Indicates some additional properties for dapr client type
+   */
+  daprProperties?: DaprProperties;
+}
+
+/**
+ * The azure resource properties
+ */
+@discriminator("type")
+model AzureResourcePropertiesBase {
+  /**
+   * The azure resource type.
+   */
+  type: AzureResourceType;
+}
+
+/**
+ * The azure resource info when target service type is AzureResource
+ */
+model AzureResource extends TargetServiceBase {
+  /**
+   * The Id of azure resource.
+   */
+  id?: string;
+
+  /**
+   * The azure resource connection related properties.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-nullable" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  resourceProperties?: AzureResourcePropertiesBase | null;
+
+  /**
+   * The target service type.
+   */
+  type: "AzureResource";
+}
+
+/**
+ * The resource properties when type is Azure Key Vault
+ */
+model AzureKeyVaultProperties extends AzureResourcePropertiesBase {
+  /**
+   * True if connect via Kubernetes CSI Driver.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-nullable" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  connectAsKubernetesCsiDriver?: boolean | null;
```

Comment:
@MaryGao Do we accept the change here?
<img width="677" height="122" alt="image" src="https://github.com/user-attachments/assets/1b452d23-e637-4a59-91d8-ea8d8c293dbd" />

Reply:
issue track: https://github.com/Azure/autorest.typescript/issues/3394

Reply:
After you have a clear discussion result, I hope you can reply under this comment to help me fix the issues in the tsp in a timely manner.

Reply:
@v-jiaodi I see in the discussion of another issue that this change seems to have already been accepted. Do you agree that we can close this comment now?